### ISSUE TITLE
Landscape/Environmental Scans

### ISSUE THREAD

**Issue by RJP43** [publishing-bitbytebit/PlainTxtPodcast#13]
## DH 501 Assignment  
What’s at stake in building your project? As you begin designing and building your research project, examine some existing DH projects that are new to you in order to think through that project’s background, mission, and context. You should locate a project - chose: [_Note to Self_ podcast](https://www.wnycstudios.org/shows/notetoself/episodes) - that is either topically related to yours, or one that uses a tool or platform that you’ll be working in.  
  
As you look through this project, write out brief answers to these questions:  
1.  How would you categorize this project? (A tool, a repository, a database, an archive?) How do you know?   

> A podcast -- which interestingly enough inherently encompasses a repository, an archive, a digital community, a tool, and a form of entertainment.   

2.  Who was involved in the building of this project (including PIs, librarians, graduate students, technical leads)?    

> This is an interesting question because it is not obvious who all is involved in every aspect of the podcast's production; however, the primary website does list the podcast's "team" here - https://www.wnycstudios.org/shows/notetoself/the-team.  
 
3.  What is the background and timeline of the building of this project?  

> According to [the _Note to Self_ Wikipedia page](https://en.wikipedia.org/wiki/Note_to_Self) - since the information was not available on the podcast's primary site - "Note to Self debuted on September 10, 2012 under the name New Tech City. On May 27, 2015 [New Tech City changed its name to Note to Self](https://www.wnycstudios.org/story/rename-note-self-new-tech-city), in response to listener suggestions. The name was inspired by the audience communicating to Manoush Zomorodi that they listen to the show because they are interested in the 'purposeful use of technology.' Zomorodi supposedly left the show in April 2018 to launch a new cryptocurrency supported podcast that was not affiliated with WNYC", but she still is the host of the latest episode I heard. There isn't much more information available detailing how long the design and concept took to be developed. What is clear from going thru numerous search results on the podcast's current and previous name is that there are a few broken links and outdated information that make it slightly confusing to clearly determine to who is involved in the current production of the podcast.     

4.  What is the project’s mission and does it succeed? How so? (Or what might be in development?)  

>  There are a couple of places where the podcast provides its mission. On each streaming service, on the primary website of the podcast, and several episodes start with some variation of this sentiment - "exploring the human side of technology." I think the podcast does indeed succeed in their mission mainly by staying on topic of addressing people and their use of technology as well as technologies effect on peoples' lives.  

5.  Discuss the level of transparency of the project: is there a narrative that tells you how it was built/by whom, or other obstacles or challenges the project faced?  

> While the podcast's primary website is not very transparent the multiple platform engagement lends itself well to being transparent. The presence of the host on social media and the podcast's engagement with multiple streaming services offers some sense of the challenges and concerns the project comes up against.  One clear example of this is how the podcast's renaming came about. Another example of this is the community engagement with the podcast's "[Projects](https://www.wnycstudios.org/shows/notetoself/projects)" = A week of interactive mini-episodes, with challenges to turn our devices from taskmasters to tools, and make positive change together."   

6.  Does the project seem “finished”? In what ways is it either finished or still developing?  

> No and yes. Yes, because the website development seems in place and complete. Also, the overall theme of the podcast seems settled on. No, because the very nature of a podcast is recursive and constantly developing as it reacts to new social trends/culture.  

7.  Discuss the chosen platform of the project, and the technology it uses (as best you can).  

> It is nearly impossible to determine if the podcast's website is built with a CMS or some kind of framework. There is no documentation indicating such information either. The site seems to use its own media player instead of embedding one of its streaming services' media players. As with all podcasts, there is an RSS feed, but I could not find site-hosted transcriptions of the episodes. There is a list of all the other platforms the podcast is available on that states ["Subscribe now to the podcast on Apple Podcasts, Pocket Casts, Stitcher, TuneIn, Spotify, Google Play Music, or wherever you get your audio."](https://www.wnycstudios.org/shows/notetoself/about) 

8.  Who appears to be the user community?  

> Podcast listeners, technology enthusiasts, technology novices, tech-curious people. There is an interesting bit in the studio's term of service documentation that states ["The digital platforms are intended for users over the age of 13."](https://www.wnyc.org/terms/). 

9.  Can you find scholarship that cites this project? How does it do so?  

> The only places I was able to find references to the podcast were Wikipedia, NPR, and in a few Medium articles written by show-affiliated people. Generally, the project is cited with very promotional language seemingly with the intention to gain more listeners.  

10.  What arguments—implicit or explicit—does the project as a whole seem to be making or advancing?

> That more people need to better understand the technologies they are interacting with daily. 

11.  How does the project promote itself?  

>  In addition to being available on nearly every podcast streaming service the website boasts the following info. as well: "Our Wednesday morning newsletter has alerts on fresh episodes, the team’s must-reads, and all the news you need to get just a little geeky. Never spammy, always thoughtful. You should sign up.  Chat with us `@notetoself` and `@manoushz` on Twitter, `@NotetoSelfRadio` on Facebook, `@manoushz` on Instagram and at `notetoself@wnyc.org`. We love to hear from you. Responses from real humans, not bots. Promise."  

12.  What’s the user experience like as you click through the site? Are there multiple pathways? Is it logical? Is it easy to read? Comment on the design as it relates to usability.  

> The primary website of the podcast's hosting studio is not very usable. I have yet to be able to find the episode transcripts, the images lack alt text, and the overal design of the site doesn't jive well with the creative vibe you get from the podcast host and the podcast's presense as seen on streaming applications.  

13.  What’s at stake in building this project? What intervention does it make in its field? What kinds of knowledge is it contributing? What work remains to be done? Where is it most successful? 

>  Unlike most podcasts which seem to just funnel information to people as listening entertainment, _Note to Self_ successfully engages with its audience through its multiple social media accounts and by offering special mini-episode project challenges that encourage listeners to take action in order to become better technology users.

**Comment by RJP43** [comment]
# _Note to Self_ with [Manoush Zomorodi](https://www.wnycstudios.org/people/manoush-zomorodi)
## Produced by [WNYC Studios](https://www.wnyc.org/)
!['note to self' written to the left of an illustration of a human or human-like figure with colorful explosions of vibrant patterns coming from the top of their head in place of hair/top of head. Snake-like explosive pattern reaches across the span of the logo above the text.](https://media.wnyc.org/i/1400/1400/l/80/1/NoteToSelf_1400X1400_TIt7xNg.png "Note to Self's logo by artist John Hersey")
`Image sourced from:` [Note to Self on www.wnycstudios.org](https://www.wnycstudios.org/shows/notetoself) `Illustration created by:` [John Hersey](http://www.john-hersey.com/)   
  
## Why this Podcast?  
Swept up in the production of the podcast's website, other graduate school responsibilites, a general self-loathing with regards to my writing abilities, and a slight embarrassment of how I came to choose this podcast -- I've delayed this scan for months. To understand why I am writing a scan on _Note to Self_  I think its important to understand how all of these reasons for delay have led me to writing on this particular podcast today `(ooo a rhyme` 😉`)`.  You see I work as a digital editor on an XML-based digital edition. What that means in regards to this podcast is I am consumed daily with angle brackets and the inner workings of what is behind-the-display the public interact with online.

**Comment by RJP43** [comment]
Here is a slightly sad realization I had when creating the above environment scan that I want to take a moment to explore since it helps highlight how I plan on making the  _plain.txt podcast_ different than what seems to be the case for many other online resources. When adding the above image of the _Note to Self_ logo I was a little stumped when it came to the task of writing the image alt text. So I went to the podcast's main site to see what information the podcast website developer(s) included. Here is what I found: 
![screenshot showing the podcast's logo has no visible alt text on the WYNC website](https://user-images.githubusercontent.com/8645521/54484163-69aaaf80-482e-11e9-84d2-959294d6f1db.png)  
Frustrated I searched for other places where the podcast is hosted on a website. Landing on the NPR page I inspected the pages and found this:  
![screenshot of NPR website's underlying HTML with alt text on the logo image reading 'Note to Self'](https://user-images.githubusercontent.com/8645521/54484175-b5f5ef80-482e-11e9-9431-15e160eb3c04.png)  Here is the alt text I ended up writing for the podcast's beautiful logo -
